full-moon/src/ast/luau.rs

```diff
@@ -19,6 +19,8 @@ pub enum TypeInfo {
     Array {
         /// The braces (`{}`) containing the type info.
         braces: ContainedSpan,
+        /// The access modifer of the array, `read` in `{ read number }`.
+        access: Option<TokenReference>,
```

Comment:
Add `#[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]`

Reply:
You didn't do this

Reply:
I added `serde_option` or whatever it is to the entire enum. If you'd rather have this, I can do that instead.

Reply:
Do you mean `display_option`? That's for display: the thing I gave is to not include it in serde if it's unset.